apps/fabric-website-resources/index.html

```diff
@@ -21,11 +21,23 @@
         // }
       };
 
-      var isLocalHost = document.location.href.indexOf('localhost') !== -1 || document.location.href.indexOf('127.0.0.1') !== -1;
+      function getParameterByName(name, url) {
+        if (!url) {
+          url = window.location.href;
```

Comment:
Should probably just use the `window.location.search` like the code you took out

Reply:
Trouble is that won't work with hash routing, if the ? is after the # (which is technically not a valid URL but likely to happen anyway)